salt/runners/pkg.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+'''
+Package helper functions using ``salt.modules.pkg``
+
+.. versionadded:: FIXME
```

Comment:
Set it to the next version codename found in salt/version.py, so right now it is `Beryllium`
https://github.com/saltstack/salt/blob/develop/salt/version.py#L81